.github/workflows/ci.yml

```diff
@@ -6,10 +6,10 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
     env:
       TEST_AUTH_TOKEN: ${{ secrets.ALPHA_TEST_AUTH_TOKEN }}
-      TEST_CACHE_NAME: dummy
+      TEST_CACHE_NAME: integration-test
```

Comment:
Might be worth making this like `js-integration-test-ci` so that it's clearer which of our sdk tests this is coming from?